content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15.md

```diff
@@ -0,0 +1,383 @@
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Determine which version to upgrade to
+
+1.  Find the latest stable 1.15 version:
+
+    {{< tabs name="k8s_install_versions" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    apt update
+    apt-cache policy kubeadm
+    # find the latest 1.15 version in the list
+    # it should look like 1.15.x-00, where x is the latest patch
+    {{% /tab %}}
+    {{% tab name="CentOS, RHEL or Fedora" %}}
+    yum list --showduplicates kubeadm --disableexcludes=kubernetes
+    # find the latest 1.15 version in the list
+    # it should look like 1.15.x-0, where x is the latest patch
+    {{% /tab %}}
+    {{< /tabs >}}
+
+## Upgrade the first control plane node
+
+1.  On your first control plane node, upgrade kubeadm:
+
+    {{< tabs name="k8s_install_kubeadm_first_cp" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.15.x-00 with the latest patch version
+    apt-mark unhold kubeadm && \
+    apt-get update && apt-get install -y kubeadm=1.15.x-00 && \
+    apt-mark hold kubeadm
+    {{% /tab %}}
+    {{% tab name="CentOS, RHEL or Fedora" %}}
+    # replace x in 1.15.x-0 with the latest patch version
+    yum install -y kubeadm-1.15.x-0 --disableexcludes=kubernetes
+    {{% /tab %}}
+    {{< /tabs >}}
+
+1.  Verify that the download works and has the expected version:
+
+    ```shell
+    kubeadm version
+    ```
+
+1.  On the control plane node, run:
+
+    ```shell
+    sudo kubeadm upgrade plan
+    ```
+
+    You should see output similar to this:
+
+    ```shell
+    [upgrade/config] Making sure the configuration is correct:
+    [upgrade/config] Reading configuration from the cluster...
+    [upgrade/config] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -oyaml'
+    [preflight] Running pre-flight checks.
+    [upgrade] Making sure the cluster is healthy:
+    [upgrade] Fetching available versions to upgrade to
+    [upgrade/versions] Cluster version: v1.14.2
+    [upgrade/versions] kubeadm version: v1.15.0
+
+    Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
+    COMPONENT   CURRENT       AVAILABLE
+    Kubelet     1 x v1.14.2   v1.15.0
+
+    Upgrade to the latest version in the v1.15 series:
+
+    COMPONENT            CURRENT   AVAILABLE
+    API Server           v1.14.2   v1.15.0
+    Controller Manager   v1.14.2   v1.15.0
+    Scheduler            v1.14.2   v1.15.0
+    Kube Proxy           v1.14.2   v1.15.0
+    CoreDNS              1.3.1     1.3.1
+    Etcd                 3.3.10    3.3.10
+
+    You can now apply the upgrade by executing the following command:
+
+        kubeadm upgrade apply v1.15.0
+
+    _____________________________________________________________________
+    ```
+
+    This command checks that your cluster can be upgraded, and fetches the versions you can upgrade to.
+
+{{< note >}}
```

Comment:
This note interrupts the numbered lists. Indenting it should keep the numbering. 

See [common shortcode issues](https://kubernetes.io/docs/contribute/style/style-guide/#common-shortcode-issues)